README.md

```diff
@@ -95,6 +121,7 @@ const api = RebillyAPI({apiKey: 'secret-api-key', sandbox: true, timeout: 10000}
 | `sandbox` | `boolean` | Flag used to enable sandbox mode for the instance. This allows you to run requests without processing real transactions on your account. Defaults to `false`. |
 | `timeout` | `integer` | Define the timeout in milliseconds for API requests. Defaults to `6000`.|
 | `version` | `string` | Define the version of the API to use. Defaults to `v2.1`. This configuration does not apply to `RebillyExperimentalAPI`. |
+| `organizationId` | `string` | Your organization identifier in scope of which need to perform request (if not specified, the default organization will be used). |
```

Comment:
https://github.com/Rebilly/RebillyAPI/pull/790/files#diff-556e74ad0df1c3a4381996298411d222R1  Isn't it `Organization-Id`?

Reply:
`Organization-Id` is a header. I set this header by `organizationId` option [here](https://github.com/Rebilly/rebilly-js-sdk/pull/187/files#diff-647808d8496ac8879120577d1444f3bdR88)

Reply:
For the request, but this table is describing the javascript object to configure the api hander. So camelCase, instead of Kebab-Case.